### ISSUE TITLE
[question] SAC entropy loss

### ISSUE THREAD

**Issue by YeeCY** [hill-a/stable-baselines#828]
The automating entropy adjustment for SAC seems a little bit different from the official `softlearning` repo. Why do we use log_ent_coef instead of the ent_coef itself, which is formulated in equation 18 of [https://arxiv.org/abs/1812.05905](https://arxiv.org/abs/1812.05905), to define the loss?

stable-baselines implementation:

https://github.com/hill-a/stable-baselines/blob/d699d6fdeaea6db24959f63f9f0de9b357f466b6/stable_baselines/sac/sac.py#L232

softlearning implementation:

https://github.com/rail-berkeley/softlearning/blob/84d7589fd5852aff9aa46debda6de39acaec2e0b/softlearning/algorithms/sac.py#L248

`
alpha_losses = -1.0 * (
                self._alpha * tf.stop_gradient(log_pis + self._target_entropy))
`

**Comment by araffin** [comment]
Duplicate of https://github.com/hill-a/stable-baselines/issues/801

**Comment by YeeCY** [comment]
OK, it seems not using log value would make any difference. They have just updated the TF2 implementation in the master branch of `softlearning`.
